site/content/en/docs/admission-check-controllers/provisioning.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Provisioning Admission Check Controller"
+date: 2023-10-23
+weight: 1
+description: >
+  An admission check controller providing kueue integration with cluster autoscaler.
+---
+
+The Provisioning Admission Check Controller is an Admission Check Controller designed to provide Kueues integration with [Kubernetes cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler). It's mainly focused on creating [ProvisioningRequests](https://github.com/kubernetes/autoscaler/blob/4872bddce2bcc5b4a5f6a3d569111c11b8a2baf4/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1/types.go#L41) for the workloads holding [Quota Reservation](docs/concepts/#quota-reservation) and keeping the [AdmissionCheckState](/docs/concepts/admission_check/#admissioncheckstate) in sync.
+
+The controller is part of kueue and configured by `ProvisioningACC` feature gate, check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide on details on feature gate configuration.
+
+## Parameters
+
+This controller uses a `ProvisioningRequestConfig` as parameters, which uses the following schema:
+
+```yaml
```

Comment:
The OpenAPI schema is not human-friendly.

Let's start with the example and follow by describing the important fields.

Reply:
done

Reply:
> What will really help is to also publish some kind of generated API documentation.

I agree. Maybe we can investigate how to do that in the next release. That would help a lot.

Reply:
https://github.com/kubernetes-sigs/kueue/issues/1249